src/components/map/MapConfig.ts

```diff
@@ -22,6 +22,9 @@ export const MapSymbology = {
         }
     },
     CountryFeature: {
+        IsHighlighted: {
+            Color: '#b0c8d4'
```

Comment:
we should add specific colours as variables with SCSS so we don't have to hardcode them in code